fern/ir-types-v22/definition/ir.yml

```diff
@@ -38,6 +38,7 @@ types:
       errorDiscriminationStrategy: ErrorDiscriminationStrategy
       sdkConfig: SdkConfig
       variables: list<variables.VariableDeclaration>
+      typeGraph: TypeGraph
```

Comment:
we'll need to make this optional to keep it in v22 (kind of annoying so if you wanted to make it required than we'd have to add it to v23).

Reply:
i could see making this type super specific for now `ServiceTypeUsageInfo` or `ServiceTypeReferenceInfo` and the docs can be about how this object

Reply:
Cool, I'll make it optional here 👍